servers/video_decoder_server.h

```diff
@@ -1,5 +1,5 @@
 /*************************************************************************/
-/*  video_stream.h                                                       */
+/*  video_decoder_server.h                                               */
```

Comment:
I think a VideoDecoderServer is for the most part unnecesary, with GDExtension, it should simply possible to provide the VideoStream<decodername> with the proper ResourceFormatLoader, both using GDExtension and that should be enough.

Reply:
It's meant to bypass the lack of ClassDB features. There's no way to check ClassDB::is_parent_class.
And I don't know how to add that either.

Reply:
ClassDB should be exposed on the GDExtension side, what check do you need to do?

Reply:
```ClassDB::is_parent_class(p_type, "VideoStream");```
The `is_parent_class` function is not exposed.
Secondly everything hinges on https://github.com/godotengine/godot-cpp/issues/652 being fixed. Currently the plugins all crash on exit due to double freeing.

Reply:
Also had a hard time registering new video formats from GDExtension. Possibly as a consequence of the Ref bug.
Trying to work in the Object* workaround.